eng/common/TestResources/New-TestResources.ps1

```diff
@@ -528,7 +533,13 @@ try {
         $templateParameters.Add('azsdkPipelineSubnetList', @($env:PoolSubnet))
     }
     # Some arm/bicep templates may want to change deployment settings (e.g. local auth) in sandboxed TME tenants
-    $templateParameters.Add('supportsSafeSecretStandard', ($context.Tenant.Name -notlike '*TME*'))
+    # The pipeline account context does not have the .Tenant.Name property, so check against subscription via
+    # naming convention instead.
+    if ($wellKnownTenantTMEStatus.Contains($TenantId)) {
```

Comment:
Perhaps a note that says TME tenants allows testing local auth and thus does not support safe secret standards.